v_3_2_1/master_template.go

```diff
@@ -277,6 +275,10 @@ write_files:
               resources:
                 requests:
                   cpu: 250m
+		  memory: 100Mi
```

Comment:
this look like bad indentation

Reply:
Not sure how github creates diff, but this is not valid the last commits